Keas.Mvc/ClientApp/components/Keys/KeyList.tsx

```diff
@@ -25,7 +26,7 @@ export default class KeyList extends React.Component<IProps, {}> {
                         <th/>
                         <th>Name</th>
                         <th>Code</th>
-                        <th className="">Available Serials</th>
+                        <th className=""><span id="serialTooltip">Available Serials<sup><i class="fas fa-info-circle"></i></sup></span><UncontrolledTooltip placement="bottom" target="serialTooltip">In Use / Total</UncontrolledTooltip></th>
```

Comment:
@jcubbage i would also probably move it so the `span` only applies to the icon, and have it open to the right instead of the bottom 
also, might as well remove `className=""`